doc/source/operations/bifrost-hardware-inventory-management.rst

```diff
@@ -0,0 +1,277 @@
+=====================================
+
+In most deployments, hardware inventory is managed by the Bifrost service.
+
+Reconfiguring Control Plane Hardware
+====================================
+
+If a server's hardware or firmware configuration is changed, it should be
+re-inspected in Bifrost before it is redeployed into service. A single server
+can be reinspected like this:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud hardware inspect --limit <Host name>
+
+.. _enrolling-new-hypervisors:
+
+Enrolling New Hypervisors
+-------------------------
+
+New hypervisors can be added to the Bifrost inventory by using its discovery
+capabilities. Assuming that new hypervisors have IPMI enabled and are
+configured to network boot on the provisioning network, the following commands
+will instruct them to PXE boot. The nodes will boot on the Ironic Python Agent
+kernel and ramdisk, which is configured to extract hardware information and
+send it to Bifrost. Note that IPMI credentials can be found in the encrypted
+file located at ``${KAYOBE_CONFIG_PATH}/secrets.yml``.
+
+.. code-block:: console
+
+   bifrost# ipmitool -I lanplus -U <ipmi username> -H <Hostname>-ipmi chassis bootdev pxe
+
+If node is are off, power them on:
+
+.. code-block:: console
+
+   bifrost# ipmitool -I lanplus -U <ipmi username> -H <Hostname>-ipmi power on
+
+If nodes is on, reset them:
+
+.. code-block:: console
+
+   bifrost# ipmitool -I lanplus -U <ipmi username> -H <Hostname>-ipmi power reset
+
+Once node have booted and have completed introspection, they should be visible
+in Bifrost:
+
+.. code-block:: console
+
+   bifrost# baremetal node list --provision-state enroll
+   +--------------------------------------+-----------------------+---------------+-------------+--------------------+-------------+
+   | UUID                                 | Name                  | Instance UUID | Power State | Provisioning State | Maintenance |
+   +--------------------------------------+-----------------------+---------------+-------------+--------------------+-------------+
+   | da0c61af-b411-41b9-8909-df2509f2059b | example-hypervisor-01 | None          | power off   | enroll             | False       |
+   +--------------------------------------+-----------------------+---------------+-------------+--------------------+-------------+
+
+After editing ``${KAYOBE_CONFIG_PATH}/overcloud.yml`` (or
+``${KAYOBE_CONFIG_PATH}/environments/${KAYOBE_ENVIRONMENT}/overcloud.yml``
+if Kayobe environment is used) to add these new hosts to
+the correct groups, import them in Kayobe's inventory with:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud inventory discover
+
+We can then provision and configure them:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud provision --limit <Hostname>
+   kayobe# kayobe overcloud host configure --limit <Hostname>
+   kayobe# kayobe overcloud service deploy --limit <Hostname> --kolla-limit <Hostname>
```

Comment:
Please add a note after this saying that you also need to reconfigure the monitoring services on the controllers, otherwise it will not show up